package.json

```diff
@@ -1,30 +1,39 @@
 {
   "name": "stylelint-config-upleveled",
-  "version": "0.0.1",
-  "description": "",
-  "main": "index.js",
-  "files": [
-    "index.js"
-  ],
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
+  "version": "1.0.1",
+  "description": "UpLeveled Stylelint defaults for styling in CSS, SCSS, Sass, CSS Modules and CSS-in-JS",
   "keywords": [
-    "stylelint",
-    "stylelint-config"
+    "config",
+    "css",
+    "sass",
+    "scss",
+    "stylelint-config",
+    "stylelint"
+  ],
+  "repository": "upleveled/stylelint-config-upleveled",
+  "license": "MIT",
+  "author": "UpLeveled",
+  "contributors": [
+    "Jose Höwer <josehower@gmail.com>",
+    "Karl Horky <karl.horky@gmail.com>"
+  ],
+  "type": "module",
+  "main": "index.cjs",
+  "files": [
+    "index.cjs",
+    "tsconfig.base.json"
```

Comment:
files to be included in npm package

Reply:
if I remove these lines then the version of the package in the npm registry is not going to contain these files which are necessary for the package to work.